k8s/namespaces/rd/rd-commondata-dataload/prod-00.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
```

Comment:
why are you deploying to prod at the same time as the other environments?

Reply:
@timja this app is not approved for production deployment yet. We created this file just for setup to make it easier during go live. Happy to remove the prod-* files, but we thought it is harmless.